framework/Makefile

```diff
@@ -40,9 +41,10 @@ $(foreach BDIR, $(BUILDIRS), $(eval $(call Add_Framework,$(BDIR))))
 
 AOBJS = $(ASRCS:.S=$(OBJEXT))
 COBJS = $(CSRCS:.c=$(OBJEXT))
+CXXOBJS = $(CXXSRCS:.cxx=$(OBJEXT))
```

Comment:
Why cxx? not cpp? What is common name for c++?

Reply:
We can support both cxx and cpp by using
`CXXOBJS = $(patsubst %.cxx,%$(OBJEXT),$(patsubst %.cpp,%$(OBJEXT),$(CXXSRCS)))`,
but what if CXXSRC have same name and different suffix?
ex) a.cpp a.cxx